### ISSUE TITLE
Wasm support 

### ISSUE THREAD

**Issue by rokk4** [flutter-stripe/flutter_stripe#1789]
Make flutter-stripe not hinder `wasm` compilation.
```
    Context: The unavailable library 'dart:html' is imported through these packages:
    
    web_plugin_registrant.dart => package:flutter_stripe_web => dart:html
    web_plugin_registrant.dart => package:flutter_stripe_web => package:stripe_js => dart:html
```

**Describe the solution you'd like**
Move away from `dart:html` and `dart:js`

> [package:web](https://pub.dev/packages/web), which replaces dart:html (and other web libraries)
> [dart:js_interop](https://api.dart.dev/stable/dart-js_interop), which replaces package:js and dart:js

https://docs.flutter.dev/platform-integration/web/wasm#js-interop-wasm

**Comment by remonh87** [comment]
we will launch a dev version soon with the new interop

**Comment by remonh87** [comment]
you can try version `6.0.0-dev.0` which I just published

**Comment by remonh87** [comment]
fixed with the release of version 6.0.0
